bin/pygrb/pygrb_make_summary_page

```diff
@@ -0,0 +1,1034 @@
+#!/usr/bin/env python
+
+# Copyright (C) 2011 Ian W. Harry
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+# =============================================================================
+# Preamble
+# =============================================================================
+
+from __future__ import division
+
+import os,sys,datetime,re,glob,shutil,ConfigParser
+from argparse import ArgumentParser
+import random
+import string
+from glue import markup
+from pylal import grbsummary,rate,antenna,git_version
+from lal.gpstime import gps_to_utc,LIGOTimeGPS
+
+__author__  = "Ian Harry <ian.harry@astro.cf.ac.uk>"
+__version__ = "git id %s" % git_version.id
+__date__    = git_version.date
+
+
+def make_external_call(\
+  command, show_stdout=False, \
+  show_command=False, show_error=True):
+  """
+  Run a command line argument and print informative messages on failure.
+  It returns two outputs: the stdout of the command, and its status.  
+
+    >>> make_external_call('cp * /tmp', False, False, True)
+
+  @param command: the command to try
+  @type command: string
+  @param show_stdout: show the stdout 
+  @type show_stdout: boolean
+  @param show_command: show the command
+  @type show_command: boolean
+  @param show_error: show the error if any
+  @type show_error: boolean
+  @return: the stdout and a status  
+
+
+  """
+  if show_command and args.verbose is True:
+    print "--- Trying this command :" + command
+
+  stdin, out, err = os.popen3(command)
+  pid, status = os.wait()
+  this_output = out.read()
+  if show_error & status != 0:
+    print >>sys.stderr, "External call failed."
+    print >>sys.stderr, "  status: %d" % status
+    print >>sys.stderr, "  stdout: %s" % this_output
+    print >>sys.stderr, "  stderr: %s" % err.read()
+    print >>sys.stderr, "  command: %s" % command
+    sys.exit(status)
+  if show_stdout:
+    if this_output[0:1]=='\n':
+      print  this_output[1:]  #first character is \n
+    else:
+      print this_output
+
+  stdin.close()
+  out.close()
+  err.close()
+  return this_output, status
+
+
+def initialize_page( title, style, script, header=None ):
```

Comment:
Move all functions like this out of the executable. I would recommend a legacy_grb.py function in the pycbc/results directory (with the desire to move over to the new output pages once everything else is finished)